index.html

```diff
@@ -105,6 +105,12 @@ <h2 class="small-name">Software Developer</h2>
     </section>
     <section  class="my-works">
```

Comment:
Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and follow best practices to write professional commit messages.